packages/web3torrent/src/components/torrent-info/channels-list/ChannelsList.tsx

```diff
@@ -47,7 +47,7 @@ function channelIdToTableRow(
   }
 
   let dataTransferred: string;
-  const peerAccount = channel[participantType];
```

Comment:
The RHS is always me, and never `peerAccount` (although it isn't super clear from the code).